earn/src/components/portfolio/LendingPairPeerCard.tsx

```diff
@@ -161,6 +164,11 @@ export default function LendingPairPeerCard(props: LendingPairPeerCardProps) {
 
   useEffect(() => {
     let mounted = true;
+    const cachedResult = cachedData.get(selectedOption.label);
+    if (cachedResult) {
+      setNumberOfUsers(cachedResult);
+      return;
+    }
     async function fetchNumberOfUsers() {
       const etherscanRequestLender0 = makeEtherscanRequest(
         7537163,
```

Comment:
Unrelated, but this should be set to 0 or have different constants for different chains.

Reply:
Good point, will address in separate PR